docs/ios/v2/how-to-guides/interact-with-room/room/polls.mdx

```diff
@@ -0,0 +1,155 @@
+---
+ title: Polls
+ nav: 7.26
+---
+
+## Introduction
+
+The Poll feature in the HMS SDK allows you to create, manage, and participate in polls and quizzes. Polls are an effective way to gather opinions, feedback, and insights from a group of people in a quick and convenient manner. With the Poll feature, you can create custom polls, set their properties, add questions, and manage responses. You can also receive real-time updates on poll status, and view poll results in an organized and easy-to-use format.
+
+This document provides guidance on how to use the Poll APIs to create a poll, receive updates on a poll, and vote on a poll.
+
+## Prerequisites
+
+- iOS SDK 0.9.6
+- Poll Read permission should be enabled for roles that can view polls
+- Poll Write and Poll Read permission should be enabled for roles that can create polls
+
+## Creating a Poll
```

Comment:
would be good to add a screenshot or pointer to the sample app where this code is written

Reply:
Added at the bottom of the doc similar to other guides